Directory.Build.props

```diff
@@ -23,6 +23,9 @@
   <PropertyGroup>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
 
+    <!-- CA1014: Mark assemblies with CLSCompliant -->
```

Comment:
Out of curiosity why this change? I might miss something but either we want to be compliant so let's fix  this or we don't care and so let's disable the rule.

Reply:
We don't care about this rule, the NoWarn is turning it off, but I can also instead move it to https://github.com/dotnet/roslyn-analyzers/blob/master/eng/Analyzers_ShippingRules.ruleset. Let me do that change.